common/sgx_workload/CMakeVariables.txt

```diff
@@ -32,4 +32,9 @@ if("${SGX_SDK} " STREQUAL " ")
     message(FATAL_ERROR "SGX_SDK environment variable not defined!")
 endif()
 
+SET(SGX_SSL "$ENV{SGX_SSL}")
+if("${SGX_SSL} " STREQUAL " ")
+    message(FATAL_ERROR "SGX_SSL environment variable not defined!")
```

Comment:
I made SGX_SSL optional with PR #125.  I suggest changing line 37 to:
```
SET(SGX_SSL "/opt/intel/sgxssl")
```

Reply:
Done